docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   akka-serverless-proxy:
-    image: gcr.io/akkaserverless-public/akkaserverless-proxy:0.8.2
+    image: gcr.io/akkaserverless-public/akkaserverless-proxy:0.8.13
```

Comment:
Some queries were not supported by 0.8.2. I guess you have another version running locally? 

Anyway, 0.8.13 is the most recent one. I do get some warnings in the logs, but didn't spend time digging in. Probably something that we can see when moving to Kalix.